server/controllers/medical/patient.js

```diff
@@ -192,7 +198,13 @@ function handleFetchPatient(uuid) {
     'ON p.debitor_uuid = d.uuid AND d.group_uuid = dg.uuid AND p.project_id = proj.id ' +
     'WHERE p.uuid = ?';
 
-  return db.exec(patientDetailQuery, [uuid]);
+  return db.exec(patientDetailQuery, uuid)
```

Comment:
Why did you need to change code in the patient controller for a cashbox pull request?